packages/core/src/Image/Image.stories.tsx

```diff
@@ -1,34 +1,271 @@
+            <DocTable
+              columns={[
+                { field: 'size', heading: 'Rounded' },
+                { field: 'useCase', heading: 'Use Case' },
+              ]}
+              data={[
+                {
+                  size: 'Default/Unset',
+                  useCase:
+                    'Use this with caution, image assets would ideally match the same ratio at twice the size for high resolution screens. ',
+                },
+                {
+                  size: 'Cover',
+                  useCase:
+                    'Most uses of images should be using cover to display images. This helps to make for a consistent front end experience that matches brand themes.',
+                },
+                {
+                  size: 'Contain',
+                  useCase:
+                    'Contain should be used for when its required that none of the image is cropped at any size or breakpoint. Images like logos and other text based images are likely examples.',
+                },
+              ]}
+            />
+            {/* <LiveDemo code={ObjectFit} /> */}
+          </Section>
+          <Section heading='Image v.s Background Image'>
+            <Text textStyle='paragraph'>
+              CSS images and background images are both used to display images on a website, but there are
+              some key differences between the two. A CSS image is an image that is inserted directly into the
+              content of an HTML page using the &lt;img&gt; tag. CSS can be used to apply styling to the
+              image, such as setting the width and height, adjusting the position, or adding borders and
+              effects. A background image, on the other hand, is an image that is used as the background for
+              an HTML element, such as a div or a section. The image is specified using CSS, and can be
+              positioned and sized in a variety of ways. Background images are often used to add visual
+              interest to a webpage or to provide a backdrop for other content. One of the main differences
+              between CSS images and background images is how they are treated by search engines and screen
+              readers. CSS images inserted using the &lt;img&gt; tag are more likely to be recognized by
+              search engines and are more accessible to screen readers, as they are considered part of the
+              content of the page. Background images, on the other hand, are often ignored by search engines
+              and may not be accessible to users with visual impairments. Another difference between the two
+              is the way they are loaded. CSS images are loaded as separate files, which can slow down the
+              page loading time if there are too many of them. Background images, on the other hand, are
+              loaded as part of the CSS file, which can improve page loading times and reduce server requests.
```

Comment:
I'd remove this as it's not technically accurate.
```
Background images, on the other hand, are
              loaded as part of the CSS file, which can improve page loading times and reduce server requests.
```

See our own `BackgroundImage` for an example: 
![CleanShot 2023-07-11 at 16 13 44](https://github.com/priceline/design-system/assets/3260642/1334d09e-b92e-42f2-b474-c83e21e96a8c)